client/gutenberg/extensions/related-posts/editor.js

```diff
@@ -0,0 +1,77 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { includes } from 'lodash';
```

Comment:
To get this block functioning, I had to change this to:
```js
import includes from 'lodash/includes';
```

Otherwise I'm seeing:
<img width="879" alt="screen shot 2018-09-19 at 11 02 18" src="https://user-images.githubusercontent.com/87168/45739935-c8cf9480-bbfc-11e8-8593-8fe6441834ba.png">

Reply:
I've heard they're going to enforce this way of importing lodash in the new Gutenberg version, guess it already happens. Will fix.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/26530/commits/73b3716c06cbd109020fca89f5342a1d3bacff7b, thanks @simison !